test/cli/commands_test.py

```diff
@@ -53,7 +53,7 @@ def invoke_assert_code(
 
 
 expected_output = """== [test/fixtures/linter/indentation_error_simple.sql] FAIL
-L:   2 | P:   4 | L003 | Indentation not hanging or a multiple of 4 spaces
+L:   2 | P:   4 | L003 | Expected 1 indentations, found 0 [compared to line 01]
```

Comment:
Have we settled on a final format for the line numbers? If we're going to use leading zeros, I would suggest allowing for 4 digits (0001). Many files will be more than 99 lines long.

Reply:
Line numbers in the spec for the start of the line default to three digits with a leading `L`. Line numbers longer than three digits automatically expand.